cmake/OpenEXRSetup.cmake

```diff
@@ -42,6 +42,11 @@ option(OPENEXR_INSTALL_PKG_CONFIG "Install OpenEXR.pc file" ON)
 # Whether to enable threading. This can be disabled, although thread pool and tasks
 # are still used, just processed immediately
 option(OPENEXR_ENABLE_THREADING "Enables threaded processing of requests" ON)
+# will use TBB for the global thread pool by default
```

Comment:
This comment's kinda confusing ~ the option is OFF (which is what I would expect) but the comment says TBB is used by default, which is only true if the option is ON.